src/client/pydaos/raw/daos_api.py

```diff
@@ -311,32 +311,6 @@ def target_query(self, tgt):
         """Query information of storage targets within a DAOS pool."""
         raise NotImplementedError("Target_query not yet implemented in C API.")
 
-    def destroy(self, force, cb_func=None):
```

Comment:
If pool destroy api is not there anymore shouldn't, the api call inside TestPool destroy() be also removed?
https://github.com/daos-stack/daos/blob/972f777b2a7b07ff230ad8cc8808121a10ad82a8/src/tests/ftest/util/test_utils_pool.py#L206

Reply:
Yes. I'll update in the next PR as currently it's not causing an issue.